addons/survey/models/survey_user.py

```diff
@@ -109,6 +109,14 @@ def _compute_attempts_number(self):
 
                 user_input.attempts_number = attempts_number
 
+    @api.model
+    def create(self, vals):
+        if 'predefined_question_ids' not in vals:
+            suvey_id = vals.get('survey_id', self.env.context.get('default_survey_id'))
+            survey = self.env['survey.survey'].browse(suvey_id)
+            vals['predefined_question_ids'] = [(6, 0, survey._prepare_answer_questions().ids)]
+        return super(SurveyUserInput, self).create(vals)
+
```

Comment:
Purposefully turning off the create_multi?

I guess we don't really need it, but it could be possible to keep it without too many issues I think.